grouper/fe/handlers.py

```diff
@@ -96,8 +98,9 @@ def get(self, user_id=None, name=None):
 
         try:
             user_md = self.graph.get_user_details(user.name)
-        except KeyError:
-            # User is probably very new, so they have no metadata yet.
+        except (KeyError, NetworkXError):
```

Comment:
Seems like we should be more explicit with the error states we're allowing.

In particular,
- Can we be more explicit than a `KeyError`? This is unrelated to this change, but we should have a specific exception if we're going to catch it, as `KeyError`s aren't specific to getting user details and might catch more exceptions than we mean to.
- Can we be more specific than a `NetworkXError`? Is there a more specific exception (or set of exceptions) that we should allow? The answer may be no, but I figured I'd ask. The comment seems very specific to disabled users but the exception handling is general.